Sources/LocalizedStringKit/LocalizedStringKit.m

```diff
@@ -115,7 +115,7 @@ + (NSBundle *)getLocalizedStringKitBundle:(NSString *_Nullable)bundleName
     // Defaults to primary bundle if bundleName not specified
     bundleName = LSKPrimaryBundleName;
   }
-  else
+  else if ([bundleName rangeOfString:@".bundle"].location == NSNotFound)
```

Comment:
```suggestion
  else if (![bundleName hasSuffix:@".bundle"])
```

Reply:
Much better! For some reason I only recalled `hasPrefix` being an API on `NSString`